PasswordManager/src/passwordmanager/PasswordClient.java

```diff
@@ -1,5 +1,107 @@
 package passwordmanager;
 
+import java.io.IOException;
+import java.net.Socket;
+
+import passwordmanager.communication.CommunicationProtocol;
+import passwordmanager.communication.CommunicationProtocol.CommunicationOperation;
+import passwordmanager.communication.CommunicationProtocol.ProtocolMode;
+import passwordmanager.communication.Query;
+import passwordmanager.communication.Response;
+import passwordmanager.config.Configuration;
+
+
+
 public class PasswordClient {
+	
+/**	
+ * 
+ *  
+ * @author Yemeri 
+ *  
+ */
+	
+/** Public classes because the PasswordController has to be able to call on them **/
+	
+	private CommunicationProtocol Protocol; 
```

Comment:
Instance variables should in general start with a lowercase character, and then use camel case for each "word" in the variable name. See Java naming conventions below:

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

See below my suggestion:

```suggestion
	private CommunicationProtocol protocol; 
```